### ISSUE TITLE
text field default value binding not refreshed

### ISSUE THREAD

**Issue by ggs331** [Budibase/budibase#5496]
Hello,

I have a option_customers binded Customer Name as label and Email as value, and I have textfield binded to this option_customers. Suppose the option_customers selected item changed, then the textfield should change as well. But on v1.0.110 and v1.0.111 didn't work. I tried bind to a paragrah it works just find, but not in text field. 

BTW, on v1.0.46 it works find. Once the options selected item changed, then text field will be synced. 

It seems something has been changed on Budibase. Or is it a bug?

![iShot2022-04-20_16 47 25](https://user-images.githubusercontent.com/29649651/164191411-6fa6dcfe-0d39-4fcd-9c7c-a95c3e191da3.gif)

**Comment by aptkingston** [comment]
Hey @ggs331. Yes we recently had to change how default values work. I'm guessing you have the text field setting for "default value" set to the binding for the other field? We had to change default values so that it only sets the value once (the first time it appears), and it no longer will update the value if default value changes. This was to fix some other bigger issues.

We know that this feature was used a lot, so we've introduced a new (and better) way of doing this. The PR for these new features is here: https://github.com/Budibase/budibase/pull/5417
We're adding the ability to execute actions whenever a field value changes, and the ability to set the value of another field. This is a more explicit and flexible way of doing things like in your example here.
